pyo3-macros-backend/src/method.rs

```diff
@@ -468,9 +489,17 @@ impl<'a> FnSpec<'a> {
                     _pyo3::impl_::coroutine::new_coroutine(
                         _pyo3::intern!(py, stringify!(#python_name)),
                         #qualname_prefix,
-                        async move { _pyo3::impl_::wrap::OkWrap::wrap(future.await) }
+                        #throw_callback,
+                        async move { _pyo3::impl_::wrap::OkWrap::wrap(future.await) },
                     )
                 }};
+                if cancel_handle.is_some() {
+                    call = quote! {{
+                        let __cancel_handle = _pyo3::coroutine::CancelHandle::new();
```

Comment:
I am not sure how we do this elsewhere, but maybe those names should include something like `__pyo3_` to have to some ad-hoc namespacing?

Reply:
There is no such kind of namespacing anywhere else. On the contrary, variables are named `function`, `output`, `_args`, `_kwnames`, etc. I was quite innovative already by using a double underscore :smile: 
Anyway, I can add this prefix if you want, I've no opinion about it.

Reply:
No, let's stay consistent then.